two command for CargoCollector

Mboesn opened the pull request:
created two commands for the CargoCollector SS
- setCargoCollector
- setCargoHolder

Mboesn committed:
two command for CargoCollector

created two commands for the CargoCollector SS
- setCargoCollector
- setCargoHolder

esh4 commented on src/main/java/frc/robot/Commands/setCargoCollector.java:
```diff
@@ -0,0 +1,51 @@
+  @Override
+  protected boolean isFinished() {
+    return Robot.cargoCollector.isHoldingBall();
+  }
```
Here's an issue that I **actually** don't have a solution for a the minute - this command and SetCargoHolder can't run at the same time because they both require(Robot.cargoCollector), which is a problem because they both **need** to run at the same time. 
Maybe the collector and holder should be seperate Subsystems?
we'll have to have a discussion about it in our next meeting...

Mboesn committed:
comments

Mboesn committed:
bug

EliyaFishman committed:
edited a bit

added the holding wheels to the first command (CollectCargo)
and added set timeout to the other command (CargoPush)

Mboesn committed:
Merge branch 'CargoCollectorCommands' of https://github.com/Programming-Trigon/2019RobotCode into CargoCollectorCommands

esh4 commented on src/main/java/frc/robot/Commands/CollectCargo.java:
```diff
@@ -30,6 +30,7 @@ protected void initialize() {
   protected void execute() {
     Robot.cargoCollector.setCargoCollectorMotor(power);
+    Robot.cargoCollector.setCargoHolderMotors(power);
   }
```
Both systems won't use the same power.

EliyaFishman committed:
added power1

added power1 so that the two functions don't use the same power

Mboesn committed:
Fixed the code according to the pull request

Mboesn committed:
Merge branch 'CargoCollectorCommands' of https://github.com/Programming-Trigon/2019RobotCode into CargoCollectorCommands

esh4 committed:
Merge branch 'master' into CargoCollectorCommands
